appveyor.yml

```diff
@@ -1,6 +1,5 @@
 platform:
   - x86
-  - x64
```

Comment:
The current plan is to only ship 32-bit on Windows because we can only manage updates for one architecture. I guess we could keep testing on 64-bit and not push builds there but I don't think it likely we'll hit cross-architecture issues.